Gemfile

```diff
@@ -63,10 +62,10 @@ end
 if facterversion = ENV['FACTER_GEM_VERSION']
   gem 'facter', facterversion.to_s, :require => false, :groups => [:test]
 else
-  gem 'facter', :require => false, :groups => [:test]
+gem 'facter', :require => false, :groups => [:test]
```

Comment:
the identation was fine here

Reply:
@tampakrap hrm, that does look wrong.
However, it will need to be fixed upstream:
https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/Gemfile.erb#L58

Reply:
referenced in https://github.com/voxpupuli/modulesync_config/pull/377